CWE/events/East Turkestan Additions.txt

```diff
@@ -0,0 +1,1404 @@
+		any_pop = {
+			limit = { OR = { culture = chinese culture = hui }
+			ideology = { factor = 0.10 value = nationalist }
+			militancy = 3
+			consciousness = 3
+		}
+		ai_chance = { 
+			factor = 0.2
+		}
+	}
+}
+
+#Dismantling the reforms
+country_event = {
+	id = 8023805
+	title = "Dismantling the Reforms"
+	desc = EVTDESC8023805
+	picture = "xij_kmt"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = XIJ
+		year = 1947
+		NOT = { year = 1950 }
+
+		is_vassal = no
+		war = no
+
+		OR = { ruling_party_ideology = nationalist ruling_party_ideology = conservative }
+		NOT = { XIJ = { war_with = ETS } } #Ili Rebellion
+
+	}
+
+	mean_time_to_happen = { months = 3 }
+
+	option = {
+		name = "It is time to bring Xinjiang in line with the rest of China."
+		remove_accepted_culture = uigher
+
+		any_pop = {
+			limit = { OR = { culture = chinese culture = hui }
+			ideology = { factor = 0.10 value = nationalist }
+			militancy = 3
+			consciousness = 3
+		}
+		ai_chance = { 
+			factor = 0.7
+		}
+	}
+
+	option = {
+		name = "We will preserve what we have built."
+		add_country_modifier = { name = political_strife duration = 395 }
+		any_pop = {
+			limit = { OR = { culture = chinese culture = hui }
+			ideology = { factor = 0.10 value = socialist }
+			militancy = -3
+			consciousness = -3
+		}
+		ai_chance = { 
+			factor = 0.2
+		}
+	}
+}
+
+#Soviet Propaganda
+country_event = {
+	id = 8023806
+	title = "Continued Efforts in East Turkestan"
+	desc = EVTDESC8023806
+	picture = "nwo2_east_turkestan"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = RUS
+		year = 1947
+		NOT = { year = 1950 }
+
+		war = no
+
+		OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 }
+
+		ETS = { 
+			exists = no
+		}
+		XIJ = { 
+			exists = yes
+			government = populist_dictatorship
+			OR = { NOT = { ruling_party_ideology = socialist ruling_party_ideology = communist ruling_party_ideology = communist_social } }
+		}
+		KMT = { war_with = CHI } #Chinese Civil War ongoing
+
+	}
+
+	mean_time_to_happen = { months = 2 }
+
+	option = {
+		name = "Continue to support the Uyghur people."
+		badboy = 1
+		money = -5000
+		diplomatic_relation = { who = KMT value = -10 }
+		diplomatic_relation = { who = CHI value = -10 }
+		diplomatic_relation = { who = XIJ value = -200 }
+		XIJ = { country_event = 8023807 }
+		ai_chance = { factor = 0.9 }
+	}
+	option = {
+		name = "Leave the issue alone for now."
+		badboy = -1
+		ai_chance = { factor = 0.1 }
+	}
+}
+
+
+#Soviet Propaganda - response
+country_event = {
+	id = 8023807
+	title = "Soviet Propaganda"
+	desc = EVTDESC8023807
+	picture = "ets_rally"
+
+	is_triggered_only = yes
+
+
+	option = {
+		name = "The situation threatens to descend into rebellion once more."
+		any_pop = {
+			limit = { culture = uighur }
+			ideology = { factor = 0.10 value = communist }
+			militancy = 3
+			consciousness = 3
+		}
+	}
+}
+
+
+#1947 unrest
+country_event = {
+	id = 8023808
+	title = "1947 unrest"

```

Comment:
Title should not use a year as event may fire in another year.

"$YEAR$ Unrest in $COUNTRY$"